src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -354,6 +361,17 @@ void Snippet::initOptimalPrimitiveDescriptor() {
         SNIPPETS_REGISTER_PASS_RELATIVE(Place::After, ov::snippets::pass::MatMulToBrgemm,
                                         pass::EnforcePrecision, element::f32, element::bf16);
     }
+
+#ifdef SNIPPETS_LIBXSMM_TPP
+   if (snippetAttrs.snippet->has_domain_sensitive_ops()) {
+       SNIPPETS_REGISTER_PASS_RELATIVE(Place::Before, ov::snippets::pass::PropagatePrecision,
+                                       ov::intel_cpu::tpp::pass::EltwiseToEltwiseTPP);
+       // Note: There could be several ConvertConstantsToScalars instances in the pipeline
+       SNIPPETS_REGISTER_PASS_ABSOLUTE(Place::PipelineEnd, ov::intel_cpu::tpp::pass::ScalarToScalarTPP);
+       SNIPPETS_REGISTER_PASS_RELATIVE(Place::Before, ov::snippets::pass::PropagatePrecision,
+                                       ov::intel_cpu::tpp::pass::BrgemmToBrgemmTPP);
+   }
+#endif
```

Comment:
Maybe it'd better if we register tpp passes after usual cpu passes? I mean to move this code part to 380 line.
I think so because we may register manually `BrgemmToBrgemmTPP` before `BrgemmToBrgemmCPU`. It's more safely because if we change `BrgemmToBrgemmCPU` registration place, we may brake `BrgemmToBrgemmTPP` registration, what do you think?

Reply:
I see your point. Sure, why not)